mmv1/third_party/terraform/services/logging/resource_logging_folder_sink.go

```diff
@@ -38,6 +38,13 @@ func ResourceLoggingFolderSink() *schema.Resource {
 		Default:     false,
 		Description: `Whether or not to include children folders in the sink export. If true, logs associated with child projects are also exported; otherwise only logs relating to the provided folder are included.`,
 	}
+	schm.Schema["intercept_children"] = &schema.Schema{
+		Type:        schema.TypeBool,
+		Optional:    true,
+		ForceNew:    true,
+		Default:     false,
```

Comment:
Is this field currently returned by current API get requests on existing resources not made with it? This is potentially a breaking change -- if at plan-time the API does not return the value this will result in terraform planning to recreate any existing instances of this resource.

Reply:
Yes. I was actually planning to remove the ForceNew behavior for both IncludeChildren and this new InterceptChildren field in a subsequent PR, so I will just roll that into this PR. Will comment again once it's added.

Reply:
Ok, this is updated now with the other commit. ForceNew has been removed for the new field, and also the IncludeChildren field, which is supposed to have nearly identical behavior.